.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+jobs:
+    build:
+        name: PHP ${{ matrix.php-versions }} ${{ matrix.composer-args }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-args: ['', '--prefer-lowest']
+                allow-failure: [false]
+                include:
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--ignore-platform-reqs --no-scripts'
```

Comment:
The `--no-scripts` option here was to prevent grumphp from installing its composer plugin, as that was throwing an error in PHP 8.  It had to do with `proc_open` and the third parameter. I don't know yet what the culprit is, as `proc_open` isn't available on 3v4l.org on PHP 8.0 yet: https://3v4l.org/5h7jf so I couldn't really test the syntax without dealing with downloading a PHP 8 container locally and testing that way.  We don't use grumphp in CI though, so not really important.